### PR TITLE
Allow trailing space characters after custom label

### PR DESCRIPTION
**Pull request by pierreprinetti**
Before this change, adding or removing a label had to be done in the
last line of the comment, or the parser would interpret the newline as
part of the label.

### CODE REVIEW

File: prow/plugins/label/label.go [hunk 1]
```diff
@@ -106,6 +106,9 @@ func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string,
 		labelFilter.Insert(strings.ToLower(l))
 	}
 	for _, match := range matches {
+		for i := range match {
```
Comment (stevekuznetsov) [inline prow/plugins/label/label.go]: Seems like we only ever use `match[0]` -- why do we need to apply this to all of them?

File: prow/plugins/label/label_test.go [hunk 2]
```diff
@@ -399,6 +399,16 @@ func TestLabel(t *testing.T) {
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
 		},
+		{
+			name:                  "Add custom label with trailing space",
+			body:                  "/label orchestrator/foo ",
```
Comment (fejta) [inline prow/plugins/label/label_test.go]: From the description it sounds like your test case should include a new line

**Pull request merged**